heartbeat/monitors/active/tcp/tcp.go

```diff
@@ -91,13 +89,6 @@ func create(
 
 		epJobs, err := dialchain.MakeDialerJobs(db, scheme, eps, config.Mode,
 			func(event *beat.Event, dialer transport.Dialer, addr string) error {
-				u, err := url.Parse(fmt.Sprintf("%s://%s", scheme, addr))
-				if err != nil {
-					return err
-				}
-
-				eventext.MergeEventFields(event, common.MapStr{"url": wrappers.URLFields(u)})
```

Comment:
Trying to find out where this is now ...

Reply:
That's been moved here: https://github.com/elastic/beats/pull/10787/files/8d33ef28de0ef07f1e8d26318aba21f7c20bff89#diff-ff8455f217780e07b3802951d4451126R151

I'll open a future refactor after this is merged to move `dialchain/builder.go` into the `tcp` package. Everything in that file is actually only used by TCP, and it's confusing that there's TCP specific code that *looks* very generic in `dialchain/builder.go`.